source/_layouts/has_complimentary.liquid

```diff
@@ -6,14 +6,7 @@ layout: base
   <div class="sl-l-large-holy-grail__body">
     <div class="sl-l-large-holy-grail__main">
       <div{% unless no_container %} class="sl-l-container sl-l-container--small"{% endunless %}>
-        {% if before_introduction %}
-          {{ before_introduction | markdown }}
-        {% endif %}
-        {% if introduction %}
-          <div class="sl-c-introduction{% if center_introduction %} sl-c-introduction--center{% endif %}">
-            {{ introduction | markdown }}
-          </div>
-        {% endif %}
+        {% render 'intro', before_introduction: before_introduction, introduction: introduction, center_introduction: center_introduction %}
```

Comment:
Seems like 'include' might be simpler here? (and in the other places we're using this partial)?

Reply:
`include` is [deprecated](https://liquidjs.com/tags/include.html). We could try `{% renderFile %}`, but it only accepts an object as an arg, and LiquidJS doesn't have a good way to generate an object on the fly.